### PR TITLE
feat: update installer

### LINKED ISSUE #2534: add docs to manual for "deno install"

### PR DESCRIPTION
**Pull request by bartlomieju**
Updates installer to latest version denoland/deno_std@b13441f

Closes #2534

### CODE REVIEW

File: cli/flags.rs [hunk 1]
```diff
@@ -350,7 +350,14 @@ Demonstrates breaking the input up by space delimiter instead of by lines:
         .long_about(
 "Automatically downloads deno_installer dependencies on first run.
 
-  deno install file_server https://deno.land/std/http/file_server.ts --allow-net --allow-read",
+  deno install file_server https://deno.land/std/http/file_server.ts --allow-net --allow-read
+  deno install -d /usr/local/bin file_server https://deno.land/std/http/file_server.ts --allow-net --allow-read",
```
Comment (ry) [inline cli/flags.rs]: How about another example that has no flags:

```
deno install colors https://deno.land/std/examples/colors.ts
```

Comment (ry) [reply cli/flags.rs]: Can you mention here that the default install directory is $HOME/.deno/bin and that people need to manually add it to their path?

**Comment by bartlomieju** [comment]
Please wait, I'm writing up manual ATM

**Comment by bartlomieju** [comment]
@ry PTAL at the manual

File: website/manual.md [hunk 2]
```diff
@@ -698,6 +698,86 @@ export main() {
 }
 ```
 
+### Executable scripts
```
Comment (ry) [inline website/manual.md]: "Installing Executable Scripts"

File: website/manual.md [hunk 3]
```diff
@@ -698,6 +698,86 @@ export main() {
...
+
+Deno provides ability to easily distribute executable code via `deno install`
+command.
```
Comment (ry) [inline website/manual.md]: Install and distribute

File: website/manual.md [hunk 4]
```diff
@@ -698,6 +698,86 @@ export main() {
...
+invoke `deno` with specified permissions and CLI flags.
+
+Example:
+
+```
+$ deno install file_server https://deno.land/std/http/file_server.ts --allow-net --allow-read
+> [1/1] Compiling https://deno.land/std/http/file_server.ts
+>
+> ✅ Successfully installed file_server.
+> /Users/deno/.deno/bin/file_server
```
Comment (ry) [inline website/manual.md]: Why does the output have ">" marks in it?

File: website/manual.md [hunk 5]
```diff
@@ -698,6 +698,86 @@ export main() {
...
+$ echo 'export PATH="$HOME/.deno/bin:$PATH"' >> ~/.bashrc
+```
+
+Installation directory can be changed using `-d/--dir` flag:
+
+```
+deno install --dir /usr/local/bin prettier https://deno.land/std/prettier/main.ts --allow-write --allow-read
+```
+
+#### Flags and permissions
```
Comment (ry) [inline website/manual.md]: I don't like all these subsubsections - it makes the ToC too complex. Leave this out and write it like a single essay.

**Pull request merged**